### PR TITLE
Add ability to force a hard delete

### PR DESCRIPTION
**Pull request by alexmadjar**
Sometimes you actually want to delete stuff.  This PR adds that option

### CODE REVIEW

File: softdelete/models.py [hunk 1]
```diff
@@ -112,7 +121,7 @@ def filter(self, *args, **kwargs):
         if 'pk' in kwargs:
             qs = self.all_with_deleted().filter(*args, **kwargs)
         else:
-            qs = self._get_base_queryset().filter(*args, **kwargs)
+            qs = self.get_queryset().filter(*args, **kwargs)
```
Comment (alexmadjar) [inline softdelete/models.py:124]: also fixes this bug with `objects.filter` not being filtered

**Comment by ryochiji** [comment]
Let's add some tests.

**Comment by alexmadjar** [comment]
@ryochiji - I spent 4 hours trying to figure out how to run their tests and I still haven't been able to.

**Comment by alexmadjar** [comment]
@ryochiji - I believe we agreed not to add tests for this?

**Comment by ryochiji** [comment]
Yep, let's test in vishnu-backend for now, and revisit these tests later.

**Comment by ryochiji** [comment]
Having said that, I'm not convinced we should use `.hard_delete()` but let's discuss that in the other PR.

**Pull request merged**